pcsx2/gui/MainFrame.cpp

```diff
@@ -643,32 +644,55 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	if( cdvd )
+	wxString Source_Label[static_cast<int>(CDVD_SourceType::NumberofSources)] = {"ISO" , "CDVD" , "BIOS"};
+	wxString Reboot_Boot_Indicator = (vmIsActive) ? "Reboot" : "Boot";
+	wxString ItemLabel = Reboot_Boot_Indicator + " " + Source_Label[static_cast<int>(source)];
+	wxString Fast_Reboot_HelpText = "Reboot using fast BOOT (skips splash screens)";
+	wxString Fast_Boot_HelpText = "Use fast boot to skip PS2 startup and splash screens";
+	wxString Full_Boot_HelpText[static_cast<int>(CDVD_SourceType::NumberofSources)];
+	wxString Full_Reboot_HelpText = "Hard reset of the active VM";
```

Comment:
You removed string translations. 

I'm going to suggest you either use `thisNamingConvention` or `this_naming_convention` for variables. See  https://github.com/PCSX2/pcsx2/pull/953#issuecomment-154707005